rules/windows/file_event/sysmon_susp_clr_logs.yml

```diff
@@ -0,0 +1,27 @@
+title: Suspcious CLR Logs Creation
+id: e4b63079-6198-405c-abd7-3fe8b0ce3263
+description: Detects suspicious .NET assembly executions 
+references:
+  - https://blog.menasec.net/2019/07/interesting-difr-traces-of-net-clr.html
+date: 2020/10/12
+tags:
+  - attack.execution
+  - attack.t1059.001
+status: experimental
+author: omkar72, oscd.community
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 11
+        TargetFilename|contains:
```

Comment:
Please delete `|contains` because such an expression will not work correctly. Without the _contains_ value modifier, events will be filtered using [this logic](https://github.com/Neo23x0/sigma/wiki/Specification#general).